pkg/git/detector/detector_whitebox_test.go

```diff
@@ -75,6 +125,39 @@ func XTestGitHubDetectorWithUsernameAndPassword(t *testing.T) {
 	printBuildEnvStats(buildEnvStats)
 }
 
+func XTestGenericGitUsingSshAccessingGitHub(t *testing.T) {
```

Comment:
As I commented in the other PR, I think t.skip() would be better.
Later we can create a separate make target for such tests. But it can be a bit tricky for the tests which requires authorization like this one. For local tests we can relay on the local key from the home dir. And for CI we can create a bot account and keep the ssh key as a secret. That's OK to do it later in separate PRs.

Reply:
Fixed